src/Components/FakeTargetsOutline.fs

```diff
@@ -0,0 +1,366 @@
+          console : string
+          stopAtEntry : bool 
+          justMyCode : bool
+          requireExactSource : bool }
+
+    let activate (context : ExtensionContext) =
+        setShowTargetsOutlineForEditor window.activeTextEditor
+
+        let treeViewId = ShowInActivity.initializeAndGetId ()
+
+        window.onDidChangeActiveTextEditor.Invoke(unbox onDidChangeActiveTextEditor)
+            |> context.subscriptions.Add
+
+        refresh.event.Invoke(fun uri ->
+            if isEnabledFor uri then
+                reallyRefresh.fire(None)
+            createEmpty)
+            |> context.subscriptions.Add
+
+        workspace.onDidChangeConfiguration.Invoke(unbox onDidChangeConfiguration)
+            |> context.subscriptions.Add
+
+        commands.registerCommand("FAKE.targetsOutline.goTo", Func<obj, obj>(fun n ->
+            let line =
+                let m = unbox<Model> n
+                m.Declaration.Line
+
+            let args =
+                createObj [
+                    "lineNumber" ==> line - 1
+                    "at" ==> "center"
+                ]
+
+            vscode.commands.executeCommand("revealLine", args)
+            |> unbox
+        )) |> context.subscriptions.Add
+
+        let runFake doDebug onlySingleTarget targetName =
+            promise {
+                match currentDocument with
+                | Some scriptFile ->
+                    let scriptDir = node.path.dirname scriptFile
+                    let scriptName = node.path.basename scriptFile
+                    let! fakeRuntime = LanguageService.FakeSupport.fakeRuntime()
+                    let preArg = if onlySingleTarget then "-st" else "-t"
+                    let args =
+                        if doDebug then [| "run"; "--fsiargs"; "--debug:portable --optimize-"; scriptName; preArg; targetName |]
+                        else [| "run"; scriptName; preArg; targetName |]
+                    let cfg : RequestLaunch = 
+                        { name = "Fake Script Debugging"
+                          ``type`` = "coreclr"
+                          request = "launch"
+                          preLaunchTask = None
+                          program = fakeRuntime
+                          args = args
+                          cwd = scriptDir
+                          console = "externalTerminal"
+                          stopAtEntry = false
+                          justMyCode = false
+                          requireExactSource = false }
+                    if doDebug then
+                        let! res = debug.startDebugging(JS.undefined, unbox cfg)
+                        ()
+                    else
+                        let! dotnet = Environment.dotnet
+                        match dotnet with
+                        | None ->
+                            let! _ = window.showErrorMessage("Cannot start debugging as no dotnet runtime was found. Consider configuring one in ionide settings.")
```

Comment:
This message is for running case not debugging. `"Cannot start FAKE script... "`

Reply:
Yes indeed, nice catch!